src/elements/Merch.js

```diff
@@ -0,0 +1,225 @@
+      specific: [
+        "Brand: Spreadshirt",
+        "80% cotton / 20% polyester (heather gray is 85% cotton / 15% rayon, heather denim and charcoal gray are 60% cotton, 40% polyester)| Fabric weight: 8.3 oz (midweight)",
+        "Contrasting, complementary-colored drawstring and inner necktape",
+        "Stitched slit inside kangaroo-style pocket and a small loop inside the hood for headphone usage",
+        "Reinforced cuffs and waist",
+        "Combed-cotton exterior, fuzzy fleece interior",
+        "Imported garment, processed and printed in the U.S.A.",
+        "Size: 3XL added for colors: white, navy, black & heather grey",
+      ],
+      img: "codesydney-womens-premium-hoodie.jpg",
+      link:
+        "https://www.spreadshirt.com.au/shop/design/codesydney+womens+premium+hoodie-D5efdbfcc2051764d0ed6ec13?sellable=jwon4NnxjgunyLrAE24r-444-23",
+    },
+    {
+      title: "Code.Sydney Bag",
+      description:
+        "Perfect for students, businessmen on the go or your favorite local hipster who needs to carry his laptop with him on his vintage bike, this computer backpack has all the bells and whistles. Dedicated padded laptop storage, a guaranteed-to-keep-your-stuff-organized interior, headphone port… what more could you need?",
+      specific: [
+        "600 denier polyester canvas",
+        "Dedicated padded exterior padded laptop compartment with side-entry",
+        "Fits most 17” laptop computers. Depth is approx 7”",
+        "Front-zippered pocket with organizer panel and headphone port",
+        "Imported garment; processed and printed in the U.S.A.",
+        "Not intended for use by children 12 and under",
+      ],
+      img: "codesydney-bag.jpg",
+      link:
+        "https://www.spreadshirt.com.au/shop/design/codesydney+computer+backpack-D5efdbfcc2051764d0ed6ec13?sellable=jwon4NnxjgunyLrAE24r-1157-33",
+    },
+    {
+      title: "Code.Sydney Cap",
+      description:
+        "This 5-panel, flexfit, low-profile baseball cap is a great wear. It's perfect for teams or individuals, for the field or the street. The addition of spandex to the material makes for a form-fit, and the extras like a sewn-in sweatband and embroidered ventilation holes make this piece a smart buy.",
+      specific: [
+        "Brand: Flexfit | Product ID: 5001",
+        "97% Cotton/3% Spandex",
+        "Structured, firm front panel with low, flexible fit",
+        "5-panel cap with reinforced pro-stitching throughout",
+        "embroidered eyelets for ventilation",
+        "Matching color undervisor",
+        "Imported; processed and printed in the USA"
+      ],
+      img: "codesydney-cap.jpg",
+      link:
+        "https://www.spreadshirt.com.au/shop/design/codesydney+baseball+cap-D5efdbfcc2051764d0ed6ec13?sellable=jwon4NnxjgunyLrAE24r-129-34",
+    },
+  ];
```

Comment:
Can you please extract this out into a separate file. Something like 'merch-data-service.js'.
You should try to keep data and logic separate wherever possible. Additionally, extracting data access into a separate service means we can easily update how the data fetching is handled (if say we want to put the merch data in a table or database) in a single place no matter how many files are accessing it.

Reply:
Hi @QjStout, how about I am going to set up the rudex for the entire site. And put all data of different pages into a single json file, and using axio to fetch. Later on, it will be much easier to transfer to database?

Reply:
Yeah that sounds good

Reply:
Or maybe redux is not necessary. Let just put all data into a single jason file?

Reply:
Make one file called something like api.service.js or data.service.js or something. All pages will fetch the data through that. Then how you want to handle the actual data for now is up to you but put all the logic inside the service file

Reply:
Currently, there is data in Volunteer page and Merch Page. Do you want me to transfer the projects section in home page to static render and put those data to the files as well?

Reply:
Yeah but make a separate PR for it if that's alright

Reply:
Cool mate, I'll do it.